src/share/poudriere/common.sh

```diff
@@ -2090,7 +2090,7 @@ _real_build_port() {
 		# Don't need to install if only making packages and not
 		# testing.
 		[ -n "${PORTTESTING}" ] && \
-		    install_order="${install_order} install-mtree install"
+		    install_order="${install_order} test install-mtree install"
```

Comment:
Oh. Stage and install are messed up and ```install``` has different meaning from inside and outside the port. I assume the later is meant here, so the code is correct.